src/actions/onboardingActions.js

```diff
@@ -385,6 +384,7 @@ export const checkUsernameAvailabilityAction = (username: string) => {
 
     // if user is offline then proceed with local registration
     if (!getState()?.session?.data?.isOnline) {
+      Toast.closeAll(); // close current ones to not block with multiple toasts
```

Comment:
🤔  I'm not getting multiple modals anymore 👍  but they're jumping every time I enter a letter and it's still covering input a bit. Can we show this toast just once? If that's hard, let's leave as it is.

![Screenshot 2020-10-07 at 16 46 22](https://user-images.githubusercontent.com/6280457/95346930-a3358680-08bc-11eb-802d-34ccb7143a22.png)

Reply:
Thanks, solved